README.md

```diff
@@ -181,6 +179,9 @@ If you want to install all services in editable mode, as well as the dev-depende
 ```bash
 make install-dev
 ```
+When using the `make install-dev` command it is important to prevent `poetry` from creating a different environment for every package.
+This can be achieved by running `poetry config virtualenvs.create false`, but there are multiple way to achieve this, it is best to consult the [poetry docs](https://python-poetry.org/docs/configuration/) to see which option fits best for you.
```

Comment:
```suggestion
This can be achieved by running `poetry config virtualenvs.create false`, but there are multiple way to achieve this. You can check the [`poetry` docs](https://python-poetry.org/docs/configuration/) to see which option fits best for you.
```